exercises/isbn-verifier/README.md

```diff
@@ -37,8 +37,6 @@ Now, it's even trickier since the check digit of an ISBN-10 may be 'X' (represen
 
 ## Bonus tasks
 
-* Implement `show(io::IO, isbn::ISBN)` to print the ISBN in the standard format with dashes.
```

Comment:
This needs to go into hints or some other local file.

Reply:
I see, yes that is true... I see how we can do that

Reply:
Putting it in the hints file should work, I think